app/src/main/java/nerd/tuxmobil/fahrplan/congress/schedule/MainActivity.kt

```diff
@@ -229,6 +230,12 @@ class MainActivity : BaseActivity(),
         if (sidePaneView != null && isFavoritesInSidePane) {
             sidePaneView.isVisible = !isScreenLocked
         }
+
+        requestedOrientation = if (AppRepository.readIsScheduleLandscape()) {
+            ActivityInfo.SCREEN_ORIENTATION_LANDSCAPE
+        } else {
+            ActivityInfo.SCREEN_ORIENTATION_UNSPECIFIED
+        }
```

Comment:
Did you consider using the same mechanism which is used for the other setting items? See `MainActivity#onActivityResult`.

Reply:
To my understanding that doesn't work.
The `onActivityResult()` is triggered as a

Reply:
I think it is no longer needed. Once I leave the settings screen `onResume` immediately updates the main screen.